pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2370,4 +2370,11 @@ self: super: {
   } super.postgrest));
 
   html-charset = dontCheck super.html-charset;
+
+  # true-name-0.1.0.4 has been tagged, but has not been released to Hackage.
+  # Also, beyond 0.1.0.4 an additional patch is required to make true-name
+  # compatible with current versions of template-haskell
+  # https://github.com/liyang/true-name/pull/4
+  true-name = appendPatch ./patches/true-name.patch
```

Comment:
We try not to carry around patches in Nixpkgs if at all possible.

Could you fetch this patch with `fetchpatch`?  There should be a bunch of examples of that in this file.

Reply:
I was hesitant to fetch this as a patch the patch for https://github.com/liyang/true-name/pull/4 is not sufficient to bring the state of the sources up to date, and it felt a bit weird to include a patch for a range of commits that spans different repositories, and which could potentially be broken by a branch being deleted from a forked repo; using an explicit patch file seemed safer in this particular case, but I can change it to a fetchpatch. I had actually originally implemented this using fetchpatch, but I also encountered problems because of a conflict between line endings, which I resolved by creating the manual patch file, but I can try to figure out how to fix that as part of the fetch.